Dockerfile

```diff
@@ -81,11 +72,5 @@ RUN curl -jkSL -o $CATALINA_HOME/lib/marlin.jar https://github.com/bourgesl/marl
     curl -jkSL -o $CATALINA_HOME/lib/marlin-sun-java2d.jar https://github.com/bourgesl/marlin-renderer/releases/download/v$MARLIN_TAG/marlin-$MARLIN_VERSION-Unsafe-sun-java2d.jar
 
 # cleanup
-RUN apk del curl && \
-    rm -rf /tmp/* /var/cache/apk/*
-
-COPY startup.sh /opt/startup.sh
-
-ENTRYPOINT /opt/startup.sh
-
-WORKDIR /opt
```

Comment:
Please readd since the current workdir (`$GEOSERVER_LIB_DIR`) might become confusing.